lib/engine/g_1849/stock_market.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require_relative 'stock_market'
+
+module Engine
+  module G1849
+    class StockMarket < StockMarket
+      attr_writer :game
```

Comment:
i don't think this is necessary, was this just added for line 30?

Reply:
The market needs access to `game` to write to the log and also to get the current phase information. Setting it in a writer was simpler than changing the signature of `initialize` for this subclass.